study.md

```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+007.to_s.ljust(3, "0")
```

Comment:
007.to_s.reverse! results in an output of "7" and not "700" since irb and pry strip the leading zeros of the integer.